examples/markers.ipynb

```diff
@@ -2,9 +2,17 @@
  "cells": [
```

Comment:
can you remove the `markers.ipynb` and replace with additional script examples if necessary